selftests/unit/test_test.py

```diff
@@ -51,8 +51,10 @@ def tearDown(self):
     def test_ugly_name(self):
         def run(name, path_name):
             """ Initialize test and check the dirs were created """
+            config = {"run.test_runner": 'runner'}
```

Comment:
Same here, maybe we need to adapt those tests to use the default runner (nrunner).

Reply:
IIUTC those tests are for the legacy features and when we will completely remove the legacy code then those tests will be removed too.